vhdbuilder/packer/generate-windows-vhd-configuration.ps1

```diff
@@ -169,7 +169,8 @@ $global:map = @{
         "https://acs-mirror.azureedge.net/aks/windows/cse/aks-windows-cse-scripts-v0.0.20.zip",
```

Comment:
@AbelHu - should we clean up any of these older (< v0.0.22) ones?

Reply:
Please keep them for now since the latest version in production is still v0.0.22.